### ISSUE TITLE
Support more than one Number of Users with Swarm

### ISSUE THREAD

**Issue by priyeshkpandey** [anhldbk/swarm#4]
Code works fine and collects statistics only if I provide Number of Users as 1 and Hatch Rate as 1.

<img width="1440" alt="cron_registerios_01" src="https://user-images.githubusercontent.com/13195191/43448202-f4c6a9f6-94ca-11e8-9b70-e2ff13b6ec3d.png">
<img width="1440" alt="cron_registerios_02" src="https://user-images.githubusercontent.com/13195191/43448200-f4965de6-94ca-11e8-8a39-48ebdfd2f507.png">
<img width="1440" alt="locustrunner_01" src="https://user-images.githubusercontent.com/13195191/43448199-f465a30e-94ca-11e8-99b8-ad582fd91796.png">
<img width="1440" alt="locustrunner_02" src="https://user-images.githubusercontent.com/13195191/43448197-f43359f8-94ca-11e8-8fec-d44ca41f432d.png">

**Comment by anhldbk** [comment]
@priyeshkpandey Please submit your log

**Comment by priyeshkpandey** [comment]
@anhldbk Looks like the issue is related to number of threads supported by the hardware. I tried with Number of Users as 10 and it worked. Previously I had tried with 60 which didn't work, I am still experimenting by varying number of threads. I will retry some more time with 10 (or whichever is the max number that works) as number of users and if it works fine all the time we can close this issue.
